api/src/main/java/pl/kolendateam/dadcard/characterCard/dto/CharacterDTO.java

```diff
@@ -30,29 +35,38 @@ public class CharacterDTO {
     public ArmorClassDTO armorClass;
     public SavingThrowDTO savingThrows;
     public double skillPoints;
-    public ArrayList <SkillsDTO> skillsList;
+    public ArrayList<SkillsDTO> skillsList;
+    public ArrayList<FeatsDTO> featsList;
 
     public CharacterDTO(Character character) {
         this.characterName = character.getCharacterName();
         this.playerName = character.getPlayerName();
         this.race = character.getRace();
         this.subRace = character.getSubRace();
+        this.size = character.getSize().getSize();
```

Comment:
jak coś jest fasadą to podaj od razu wartość, a nie znajomego który ma dopiero tą wartość

Reply:
poprawiłem